common/achievements/ethogirl.ts

```diff
@@ -0,0 +1,50 @@
+import EthosLabCommon from '../cards/hermits/ethoslab-common'
+import EthosLabRare from '../cards/hermits/ethoslab-rare'
+import EthosLabUltraRare from '../cards/hermits/ethoslab-ultra-rare'
+import ShadEECommon from '../cards/hermits/shadee-common'
+import ShadeEERare from '../cards/hermits/shadeee-rare'
+import {Hermit} from '../cards/types'
+import {SlotComponent} from '../components'
+import query from '../components/query'
+import {achievement} from './defaults'
+import {Achievement} from './types'
+
+const ETHO_CARDS = [
+	EthosLabCommon,
+	EthosLabRare,
+	EthosLabUltraRare,
+	ShadEECommon,
+	ShadeEERare,
+]
```

Comment:
This will make the contains check more pretty
```suggestion
const ETHO_CARDS = [
	EthosLabCommon,
	EthosLabRare,
	EthosLabUltraRare,
	ShadEECommon,
	ShadeEERare,
].map(card => card.props.id)
```